app/helpers/format-duration.js

```diff
@@ -0,0 +1,17 @@
+import { helper } from '@ember/component/helper';
+import moment from 'moment';
+
+export function formatPrice(params/*, hash*/) {
+	let duration = parseInt(params[0]);
+	if(isNaN(duration) || duration == 0){
+			return "--";
+		}
+		else{
+			let res = moment.duration(duration)._data;
+      return (res.hours?`0${res.hours}`.slice(-2) + ':':'00:')
```

Comment:
See https://runkit.com/embed/rpv3cpk8opia

Reply:
Its the same, I have just added formatting over it.

Reply:
`._data` is private API. Should avoid using it if we can.
can't we just : https://runkit.com/abhishek97/moment-example